official/vision/beta/projects/yolo/modeling/tests/test_CSPConnect.py

```diff
@@ -0,0 +1,54 @@
+import tensorflow as tf
+import tensorflow.keras as ks
+import numpy as np
+from absl.testing import parameterized
+
+from official.vision.beta.projects.yolo.modeling.building_blocks import CSPDownSample as layer
+from official.vision.beta.projects.yolo.modeling.building_blocks import CSPConnect as layer_companion
```

Comment:
Please import modules and do not import classes. It is for google py readability.

Reply:
fixed in [this commit](https://github.com/tensorflow/models/pull/9405/commits/8ee48095b1e4918478720c688e7703764bffd457)